workshop/content/20-typescript/30-hello-cdk/300-cdk-watch.md

```diff
@@ -0,0 +1,179 @@
++++
+title = "CDK Watch"
+weight = 300
++++
+
+## Faster Personal Deployments
+
+{{% notice info %}} This section is not necessary to complete the workshop, but we
+recommend that you take the time to see how `cdk watch` can speed up your personal
+deployments.
+{{% /notice %}}
+
+It's great that we have a working lambda! But what if you want to tweak the lambda
+code to get it just right? Let's say that we have now decided that we want our
+lambda function to respond with `"Good Morning, CDK!"` instead of `"Hello, CDK"`.
+
+So far, it seems like the only tool we have at our disposal to update our stack is
+`cdk deploy`. But `cdk deploy` takes time; it has to deploy your CloudFormation
+stack and upload the `lambda` directory from your disk to the boostrap bucket. If
+we're just changing our lambda code, we don't actually need to update the
+CloudFormation stack, so that part of `cdk deploy` is wasted effort.
+
+We really only need to update our lambda code. It would be great if we had
+some other mechanism for doing only that...
+
+## See how long 'CDK Deploy' takes
+
+First, as an exercise, change your lambda code in `lambda/hello.js` slightly and 
+time how long it takes to run `cdk deploy`. It will help us get a baseline for how 
+long `cdk deploy` takes:
+
+```bash
+time cdk deploy
+```
+
+The output will look something like this:
+
+```
+INSERT OUTPUT HERE
+```
+
+## CDK Watch
+
+{{% notice info %}} This command deliberately introduces drift in CloudFormation 
+stacks in order to speed up deployments. For this reason, only use it for 
+development purposes. Never use CDK Watch for your production deployments!
+{{% /notice %}}
```

Comment:
This is actually only a feature of hotswapping, not of `cdk watch`, and you can use "watch" without hotswapping (`cdk watch --no-hotswap`).

We need to make sure the distinction between the two is clearly explained here.

Reply:
do we need to add more beyond what Calvin has already suggested re: `--hotswap` flag and `--no-hotswap` option?

Reply:
I think that's enough info there.

(I'll probably change my mind once I see it all written out, but for now it's good enough 😛)